testsuite/features/reposync/srv_sync_products.feature

```diff
@@ -186,29 +188,30 @@ Feature: Synchronize products in the products page of the Setup Wizard
     And I wait until I do not see "Loading" text
     And I enter "SUSE Linux Enterprise Server 15 SP7" as the filtered product description
     When I open the sub-list of the product "SUSE Linux Enterprise Server 15 SP7 x86_64"
-    And I select "SUSE Manager Proxy Extension 5.1 x86_64" as a product
-    Then I should see the "SUSE Manager Proxy Extension 5.1 x86_64" selected
+    And I select "SUSE Multi-Linux Manager Proxy Extension 5.1 x86_64 (BETA)" as a product
+    Then I should see the "SUSE Multi-Linux Manager Proxy Extension 5.1 x86_64 (BETA)" selected
     When I click the Add Product button
     And I wait until I see "Selected channels/products were scheduled successfully for syncing." text
-    And I wait until I see "SUSE Manager Proxy Extension 5.1 x86_64" product has been added
-    And I wait until all synchronized channels for "suma-proxy-extension-51-sp7" have finished
+    And I wait until I see "SUSE Multi-Linux Manager Proxy Extension 5.1 x86_64 (BETA)" product has been added
+    And I wait until all synchronized channels for "suse-multi-linux-manager-proxy-51-sp7" have finished
 
 @proxy
 @susemanager
 @transactional_server
-  Scenario: Add SUSE Manager Retail Branch Server Extension 5.0 with SLE Micro 5.5 as base OS
+  Scenario: Add SUSE Manager Retail Branch Server Extension 5.1 with SL Micro 6.1 as base OS
```

Comment:
This misses the rebrand

Reply:
done also in a couple of other scenario names